bindings/c/Cargo.toml

```diff
@@ -19,11 +19,11 @@
 name = "opendal-c"
 publish = false
 
-authors = ["OpenDAL Contributors <dev@opendal.apache.org>"]
+authors = ["Apache OpenDAL <dev@opendal.apache.org>"]
```

Comment:
Is this a common practice?  Take the project name as the author?

Reply:
https://github.com/apache/spark/blob/b5f592bdb1a1b8f78aa652915b9c506631a14659/python/setup.py#L233

Seems Spark uses `Spark Developers`, not sure about this.

Reply:
I follow what Arrow DataFusion does - https://github.com/apache/arrow-datafusion/blob/9b78efaeccd57ab8b8e20c29174a121af8130376/Cargo.toml#L25

Let's convey the Apache brand as possible.

FWIW, Spark has a lot of branding issues ..